voronoi.cpp

```diff
@@ -143,19 +143,18 @@ void Voronoi::relax_points(int iterations = 1) {
 	std::vector<jcv_point> new_points;
 	new_points.reserve(_points.size());
 	for (int j = 0; j < iterations; j++) {
-		Ref<VoronoiDiagram> diagram { memnew(VoronoiDiagram) };
-		jcv_diagram_generate_useralloc(
+		jcv_diagram diagram;
+		memset(&diagram, 0, sizeof(jcv_diagram));
+		jcv_diagram_generate(
 			_points.size(),
 			_points.data(),
-			_boundaries.get(),
-			NULL,
-			&useralloc,
-			&userfree,
-			diagram->_diagram
+			0,
```

Comment:
Using NULL here was a little more correct (Sorry for being picky, but if there is a constant meant for that we should use it :))